docs/user-guide/query/gremlin/gremlin-limits.md

```diff
@@ -15,15 +15,13 @@ Current TinkerPop features not present in the GafferPop implementation:
 
 Current known limitations or bugs:
 
-- Proper user authentication is only available if using a Gremlin server and
-  the `GafferPopAuthoriser` class.
+- Proper [user authentication](../../../administration-guide/gaffer-deployment/gremlin.md#user-authentication)
+  is only available if using a Gremlin server and the `GafferPopAuthoriser` class.
 - Performance compared to standard Gaffer `OperationChain`s will likely be
   slower as multiple Gaffer `Operations` may utilised to perform one Gremlin
   step.
-- The ID of an Edge follows a specific format that is made up of its source and
-  destination IDs like `[source, dest]`. To use this in a seeded query you must
-  format it like `g.E("[source, dest]")` or a list like
-  `g.E(["[source1, dest1]","[source2, dest2]"])`
+- Edge IDs in GafferPop are not the same as in standard Gremlin. Instead of `g.E(11)`
+  edge IDs take the format `g.E([source, dest])` or `g.E([source, label, dest])`.
```

Comment:
Is it like this or a String like `g.E("[source, dest]")`?

Reply:
oops yes i shall fix it to be a string!